### DISCUSSION TITLE
How to hide the message "You can open bookmarks, reading mode..." using chrome options which is open when running chromedriver

### DISCUSSION THREAD

**Discussion by leeakaglaze** [giggio/node-chromedriver#484] [Q&A] [answered]
Hello.
I have one problem when using chromedriver for automation.
Sometimes some tests are failing because they can't click on any field. The problem as i observed in the following:

When chromedriver is open sometimes in Doner menu the blue hint is shown:
<img width="280" alt="Screenshot 2024-06-21 at 13 58 05" src="https://github.com/giggio/node-chromedriver/assets/137068118/0a7c9b19-31dc-4090-9688-5f83ccc10a6b">

As the result it blocks tests from finding or clicking any fields. 
The question is how to hide this message before running chromedriver itself ?

**Comment by giggio** [comment, marked as answer]
This is a Chromedriver issue, not an issue with this npm package, that only installs Chromedriver. Please direct it to the Chromedriver team on their [issue tracker](http://chromedriver.chromium.org/help).
